databricks/koalas/indexes.py

```diff
@@ -2783,6 +2805,30 @@ def get_level_values(self, level):
         )
         return ks.DataFrame(internal).index
 
+    def item(self):
+        """
+        Return the first element of the underlying data as a python tuple.
+
+        Returns
+        -------
+        tuple
+            The first element of MultiIndex.
+
+        Raises
+        ------
+        ValueError
+            If the data is not length-1.
+
+        Examples
+        --------
+        >>> kmidx = ks.MultiIndex.from_tuples([('a', 'x')])
+        >>> kmidx.item()
+        ('a', 'x')
+        """
+        if len(self._kdf.head(2)) == 1:
```

Comment:
Can you avoid executing jobs twice? calculating the length and collecting to pandas DataFrame.

Reply:
Okay! Thanks :)